model/postgres/postgres_resource.rb

```diff
@@ -104,6 +104,11 @@ module HaType
     SYNC = "sync"
   end
 
+  module Flavor
+    POSTGRES = "standard"
+    PARADEDB = "paradedb"
```

Comment:
Do you want to do this enumeration as a string, or something that is truly closest to a namespaced, singleton symbol, which the closest might be:

```ruby
[1] clover-development(main)> module Flavor
[1] clover-development(main)*   class ParadeDB; end;
[1] clover-development(main)* end;  
[2] clover-development(main)> Flavor::ParadeDB
=> Flavor::ParadeDB
```

Reply:
Keeping them string allows direct comparison with the value in the database. We can make them class but then we need to compare them to hardcoded strings somewhere and convert to class.